docs/tutorials/add-a-user_dmanis.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+---
+
+# Tutorial: Add a user
+
+This tutorial will teach you how to add a new user to the To-Do system.
```

Comment:
You're underselling your tutorial--it's doing much more!

You should have an overview that describes the big picture, and how the procedures that follow support the overall goal.